src/View/Item/header.html.twig

```diff
@@ -0,0 +1,72 @@
+<header>
+    <!--navbar-->
+    <nav>
+        <img class="navbar_img" src="assets\images\oxygen-logo.png" alt="">
+      
+        <ul class="navbar_list">
+            <li><a href="">Accueil</a></li>
+            <li><a href="">Formations</a></li>
+            <li><a href="">favoris</a></li>
+            <li><a href="">Contact</a></li>
+
+        </ul>
+    </nav>
+
+    <!--end navbar-->
+
+    <!--description and title-->
+    <div class="header__description description">
+        <h1> FORMEZ-VOUS AU METIERS <br> QUI RECRUTENT </h1>
```

Comment:
H1 should be different in every pages, find another tag for this title because it will be on every pages

Reply:
all the h or if i set h2 normal !